src/system/ExtensionsModule/Resources/docs/ExtensionsInterfaceController_and_Twig_tags.md

```diff
@@ -0,0 +1,53 @@
+ExtensionsInterfaceController
```

Comment:
no - these are DEV docs - they do not belong in any help action. Please move them to the folder I indicated.